main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-steputils/v2/stepenv"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-utils/v2/pathutil"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/step"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/xcodebuild"
+)
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	logger := log.NewLogger()
+	step := createStep(logger)
+
+	config, err := step.ProcessConfig()
+	if err != nil {
+		logger.Errorf(err.Error())
+		return 1
+	}
+
+	exitCode := 0
+	result, err := step.Run(*config)
+	if err != nil {
+		logger.TErrorf(err.Error())
```

Comment:
Question: Why use `TErrorf` here instead of `Errorf` like elsewhere?

Reply:
The only error run can produce is the error of the `xcodebuild` command run.
I wanted to make sure the command run's timeframe is logged.
- xcodebuild command start is logged with timestamp here: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/files#diff-f9b80317077cdd80a92e3996a30b0f49975f2bdb00b194cf4bab3538a0850654R125
- if it returns with an error the log is here: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R35
- in the case of a successful run it is logged here: https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/files#diff-f9b80317077cdd80a92e3996a30b0f49975f2bdb00b194cf4bab3538a0850654R148

Build log looks like this:
```
Running tests:
[09:13:29] xcodebuild "test-without-building" "-xctestrun" "/var/folders/jc/dg8f_wz562qgy5ch5wv692_r0000gn/T/deploy2127253522/BullsEye_FailingTests_iphonesimulator15.5-arm64-x86_64.xctestrun" "-destination" "platform=iOS Simulator,name=iPhone 12 Pro Max" "-resultBundlePath" "/var/folders/jc/dg8f_wz562qgy5ch5wv692_r0000gn/T/XCUITestOutput3258310731/Test-BullsEye_FailingTests_iphonesimulator15.5-arm64-x86_64.xcresult" "-retry-tests-on-failure" "-test-iterations" "3"

....

** TEST EXECUTE FAILED **

Testing started
Testing started for configuration 'Configuration 1'
[09:14:06] failing tests (exist status 65)

```

Reply:
ah ok. I didn't realize that the `T...` variants of the logger methods printed a timestamp. Looks good 👍